test/model/src/core/util.cpp

```diff
@@ -304,6 +304,40 @@ parse_uint64(const char *str, const char **end)
     return r;
 }
 
+/*
+ * wt_evict --
+ *     Evict a WiredTiger page with the given key.
+ */
+void
+wt_evict(WT_CONNECTION *conn, const char *uri, const data_value &key)
+{
+    WT_SESSION *session;
+    int ret = conn->open_session(conn, nullptr, nullptr, &session);
+    if (ret != 0)
+        throw wiredtiger_exception("Cannot open a session: ", ret);
+    wiredtiger_session_guard session_guard(session);
+
+    ret = session->begin_transaction(session, "ignore_prepare=true");
+    if (ret != 0)
+        throw wiredtiger_exception("Transaction begin failed: ", ret);
+    session_guard.rollback_on_close();
```

Comment:
I think this `rollback_on_close` mechanism works fine, but I'm not sure why it's necessary. WiredTiger will rollback any running transactions when `session->close` is called.

Reply:
Nice! I backed that out in [571b6f8](https://github.com/wiredtiger/wiredtiger/pull/10577/commits/571b6f8f95859ec3722deb0face1138b98555877).